standalone-metastore/pom.xml

```diff
@@ -71,7 +71,7 @@
     <mariadb.version>2.5.0</mariadb.version>
     <mssql.version>6.2.1.jre8</mssql.version>
     <mysql.version>8.0.31</mysql.version>
-    <postgres.version>42.5.1</postgres.version>
+    <postgres.version>42.5.6</postgres.version>
```

Comment:
https://mvnrepository.com/artifact/org.postgresql/postgresql The latest version is 42.7.3, can we upgrade it to the latest version?